core/src/main/java/org/apache/lucene/queryparser/classic/MapperQueryParser.java

```diff
@@ -779,7 +757,10 @@ private Query applySlop(Query q, int slop) {
                 builder.add(terms[i], positions[i]);
             }
             pq = builder.build();
-            pq.setBoost(q.getBoost());
+            //nocommit why do we deal with boost here? isn't that what applyBoost does?
```

Comment:
are you sure that this complication is needed? It seems like this boost business was only needed because applySlop used to be called after applyBoost. Now that we call applyBoost afterwards, we should be ok? I might be missing something though :)